examples/cs-schedules/MyStack.cs

```diff
@@ -0,0 +1,92 @@
+using Pulumi;
+using Pulumi.PulumiService;
+using Pulumi.PulumiService.Inputs;
+using Pulumi.Random;
+
+class MyStack : Stack
+{
+    public MyStack()
+    {
+        // Deployment Settings are required to be setup before schedules can be
+        // Note the `DependsOn` option in all of the schedules
+        var settings = new DeploymentSettings(
+            "Deployment Settings",
+            new DeploymentSettingsArgs{
+                Organization = "service-provider-test-org",
+                Project = "cs-schedules",
+                Stack = "test-stack",
+                SourceContext = new DeploymentSettingsSourceContextArgs{
+                    Git = new DeploymentSettingsGitSourceArgs{
+                        RepoUrl = "https://github.com/example.git",
+                        Branch = "refs/heads/main"
+                    }
+                }
+            }
+        );
+
+        // Schedule that runs drift every Sunday midnight, but does NOT remediate it
+        var drift = new DriftSchedule(
+            "drift-schedule",
+            new DriftScheduleArgs
+            {
+                Organization = "service-provider-test-org",
+                Project = "cs-schedules",
+                Stack = "test-stack",
+                ScheduleCron = "0 0 * * 0",
+                AutoRemediate = false
+            }, 
+            new CustomResourceOptions
+            {
+              DependsOn = { settings }
+            }
+        );
+
+        // Schedule to destroy stack resources on Jan 1, 2026, but NOT delete the stack itself
+        var ttl = new TtlSchedule(
+            "ttl-schedule",
+            new TtlScheduleArgs{
+                Organization = "service-provider-test-org",
+                Project = "cs-schedules",
+                Stack = "test-stack",
```

Comment:
Same idea everywhere below

```suggestion
                Organization = settings.Organization,
                Project = settings.Project,
                Stack = settings.Stack,
```